composer.json

```diff
@@ -34,6 +34,9 @@
   "extra": {
     "branch-alias": {
       "dev-master": "2.x-dev"
+    },
+    "typo3/cms": {
+      "extension-key": "fal-securedownload"
```

Comment:
This should be the extension key. so with a _ instead of -

Reply:
aad46229a6ad9a798d0f86ae93353f90ebdd6eed